PLAYER/SetPlayerMayNotEnterAnyVehicle.md

```diff
@@ -8,7 +8,8 @@ ns: PLAYER
 void SET_PLAYER_MAY_NOT_ENTER_ANY_VEHICLE(Player player);
 ```
 
+Establishes a reset flag to prevent the player from entering any vehicle. Not that this native must be called every frame.
 
 ## Parameters
-* **player**: 
+* **player**: The player index, or -1 to get the local player ped.
```

Comment:
Usage of `-1` is discouraged, usually `PlayerPedId()` is used in Lua in favor of it.